src/tools/hunspell.cxx

```diff
@@ -116,6 +116,9 @@
 #include "../parsers/odfparser.hxx"
 
 #define LIBDIR                \
+  DATADIR "/hunspell:"        \
+  DATADIR "/myspell:"         \
+  DATADIR "/myspell/dicts:"   \
   "/usr/share/hunspell:"      \
```

Comment:
I wonder if this should replace /usr/share/hunspell: etc instead of augment it, but doesn't hurt as is I guess